novawallet/Modules/Proxy/ProxyMessageSheet/ProxyMessageSheetViewFactory.swift

```diff
@@ -62,21 +80,28 @@ enum ProxyMessageSheetViewFactory {
         type: LocalizableResource<String>,
         completionCallback: @escaping MessageSheetCallback
     ) -> MessageSheetViewProtocol? {
-        let wireframe = MessageSheetWireframe()
-
-        let presenter = MessageSheetPresenter(wireframe: wireframe)
-
         let title = LocalizableResource { locale in
             R.string.localizable.proxySigningNotEnoughPermissionsTitle(preferredLanguages: locale.rLanguages)
         }
 
         let message = LocalizableResource { locale in
-            R.string.localizable.proxySigningNotEnoughPermissionsMessage(
-                proxiedName,
-                proxyName,
-                type.value(for: locale),
+            let marker = AttributedReplacementStringDecorator.marker
```

Comment:
why?

Reply:
This is how the decorator works: we first replace formats with the marker and then dermine positions of the marker to highlight replacements